src/App.jsx

```diff
@@ -24,6 +25,18 @@ const App = () => {
       updateAlert(false);
     }, 1000);
   }, []);
+  const debounce = useCallback(
```

Comment:
move this to some other util file instead of app.jsx as this could be used in other places too.

Reply:
should I create a new util file for this, as I can also include the code in arrange-json file present in utils but it will get mixed.

Reply:
hello sir, I have found an alternative way way without the use of debounce method directly passing the callback function without using debounce. What I found out is this approach provides a bit faster search rate when I search for icons as when I use debounce method it lags a bit. I am attaching screenshots for a better understanding.

Previously
![1](https://user-images.githubusercontent.com/67755381/174334215-05992bfc-7369-4996-aa86-04a78ef3a97e.jpg)
![2](https://user-images.githubusercontent.com/67755381/174334217-06f7b7ba-9d92-43fd-adfd-16f137b71645.jpg)

Now
![3](https://user-images.githubusercontent.com/67755381/174334230-ad857120-2e71-4cf9-b860-168695886d2a.jpg)

Should I implement this approach ? I was trying to create a custom hook but it was failing so I tried a different approach... @Abhirup-99 @kbhutani0001